library/core/src/alloc/global.rs

```diff
@@ -221,20 +222,21 @@ pub unsafe trait GlobalAlloc {
     ///
     /// # Safety
     ///
-    /// This function is unsafe because undefined behavior can result
-    /// if the caller does not ensure all of the following:
+    /// The caller has to ensure that
     ///
-    /// * `ptr` must be currently allocated via this allocator,
+    /// * `ptr` is allocated via this allocator,
```

Comment:
"currently allocated" is a specific term defined above, so I think this should still say "currently allocated"
```suggestion
    /// * `ptr` is currently allocated via this allocator,
```